### DISCUSSION TITLE
Transition to this dev extension from traditional Vite project.

### DISCUSSION THREAD

**Discussion by GiridharRNair** [aklinker1/vite-plugin-web-extension#174] [Q&A] [answered]
Hi, I'm considering transitioning to your dev plugin for my extension 

current state: https://github.com/GiridharRNair/ProfStatsUTD

Currently, I'm using CrxJs (https://github.com/crxjs/chrome-extension-tools) for development. Your Vite plugin appears to simplify the process. Can I integrate it into my existing project instead of starting a new one from scratch? Or is it necessary to recreate the extension using your plugin? Apologies if this question seems basic; I'm relatively new to web development.

**Comment by aklinker1** [comment, marked as answer]
No, you don't have to rewrite your project from scratch.

When I switch frameworks/build tools, I like to create basic examples with both tools so I can compare and contrast them and see their differences. If the tools have a command to bootstrap a project template, that's all I do.

Once you understand the differences, then you can bring the new tool into your real project and make all the changes necessary.

Sorry, I don't have time to help convert your project, so you're gonna have to work through any issues you run into by yourself or you can ask about them here.

I'll do what I can to help answer specific questions!
